util/test/sub_test.py

```diff
@@ -276,6 +276,15 @@ def expandvars_chpl(path):
     expand_env.update(get_chplenv())
     return string.Template(path).safe_substitute(expand_env)
 
+# Wait up to timeout seconds for files to exist. Useful on systems where file
+# IO may only complete after the launcher returns
+def WaitForFiles(files, timeout=10):
+    waited = 0
+    for f in files:
+        while not os.path.exists(f) and waited < timeout:
+            time.sleep(1)
+            waited += 1
+
```

Comment:
This doesn't care if it never sees all files right? Since then the test will fail like it did before?

Reply:
Right, if all files don't exist after the timeout we'll fail like we would before during the `cat`